kyuubi-server/src/main/scala/org/apache/kyuubi/operation/ExecuteStatement.scala

```diff
@@ -69,6 +69,7 @@ class ExecuteStatement(
       // in thrift protocol, in sync mode, we cannot distinguish between long-run query and
       // engine crash without response before socket read timeout.
       _remoteOpHandle = client.executeStatement(statement, true, queryTimeout)
+      setHasResultSet(_remoteOpHandle.isHasResultSet)
```

Comment:
Shall we also update it according to TGetOperationStatusResp in waitStatementComplete

Reply:
https://github.com/apache/incubator-kyuubi/blob/e0d9458144e68411f1dd62afb91efd1e1ef92cc5/kyuubi-common/src/main/scala/org/apache/kyuubi/service/ThriftBinaryFrontendService.scala#L415-L437

There is a unified configuration inside `GetOperationStatus` method, here we only need to change the operation's state